src/helpers/helpers.core.js

```diff
@@ -7,18 +7,20 @@ export function clampAll(obj, from, to) {
   return obj;
 }
 
-export function inPointRange(point, center, radius) {
+export function inPointRange(point, center, radius, borderWidth) {
```

Comment:
I don't think we should move the borderWidth in this helper, making it more complex than it needs to be. We have a point and we test if that is in range from the center using radius. Radius should be the full radius tested.

Reply:
the use case is the radius is 0 and borderWidth > 0. the isPointRange could return true with the previous implementation, instead, with radius = 0 should return false.

Reply:
should be `false` because the annotation is not drawn

Reply:
https://codepen.io/stockinail/pen/ExwGBZX

Click at x:5, y:5 and a popup will appear. Maybe this is a desired behavior, I don't know, but I think if the annotation is not visible, it shouldn't fire any event.

Reply:
it has display: true doesn't it? I'm not sure why we need multiple ways to hide the annotations while disabling interaction with them too.

Maybe the drawing is actually wrong? you can draw a border around a point that has zero radius.

Reply:
Looks like borderWidth is actually applied to radius? https://codepen.io/kurkle/pen/rNGoEEp

Reply:
As the drawing code is from chart.js lets live with that. So it does not draw when radius is 0.
I think you are correct that its probably unexpected that an invisible point reacts to click. But what else would be the purpose of a point that is not drawn and has `display: true`?

Reply:
I fully agree with you. Anyway also in CCS, there are always 2 attributes: enabled to enable the componenet and visible, to show or not the component.
I think display = enabled and visible, in the plugin, is something calculated automatically (borderWidth for line, radius for point and polygons, content for label)

Reply:
> But what else would be the purpose of a point that is not drawn and has `display: true`?

I think display : true declares the willing of the user to draw the annotation but it does not mean it will. 
I don't have any specific use case, anyway, if you use a callback and some specific logic, it could be that will happen, I guess.

Reply:
Anyway I'm available to revert it and to take time to think about that.

Reply:
Its consistent with the other methods this way.